.hlint.yaml

```
@@ -63,6 +63,7 @@
 - ignore: { name: Avoid lambda using `infix` }
 - ignore: { name: Avoid lambda }
 - ignore: { name: Use /= }
+- ignore: { name: Use isNothing }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)